### PR TITLE
Review the content of PyTorch tutorial

### PR DESCRIPTION
**Pull request by Ishitori**
Reviewed and extended PyTorch to MXNet tutorial

### CODE REVIEW

File: python/guide/getting-started/to-mxnet/pytorch.md [hunk 1]
```diff
@@ -25,15 +18,18 @@ For MXNet we use pip:
 # !pip install mxnet
 ```
 
-## Multidimensional matrix
+Both PyTorch and Apache MXNet have separate packages if you want to use CUDA. To install Apache MXNet with GPU support, you need to specify CUDA version. For example the snippet below will install Apache MXNet with CUDA 9.2 support:
```
Comment (sad-) [inline python/guide/getting-started/to-mxnet/pytorch.md]: sugegest removing the first sentence: ~~Both Pytorch and Apache MXNet have separate packages ...~~

File: python/guide/getting-started/to-mxnet/pytorch.md [hunk 2]
```diff
@@ -205,27 +191,24 @@ for epoch in range(5):
...
 
 ## Conclusion
 
-MXNet and PyTorch share a lot of similarities. It makes PyTorch users much easy
-to use MXNet.
+As we saw above, Apache MXNet Gluon API and PyTorch are similar in use.
+
+## Recommended Next Steps
+
+While Apache MXNet Gluon API is very similar to PyTorch, there are some extra functionality using which can make your code even faster. Check out [Hybridize tutorial](https://beta.mxnet.io/guide/packages/gluon/hybridize.html) to learn how to write imperative code which can be converted to symbolic one. Also, check out how to extend Apache MXNet with your own [custom layers](https://beta.mxnet.io/guide/extend/custom_layer.html).
```
Comment (sad-) [inline python/guide/getting-started/to-mxnet/pytorch.md]: Suggest having this as bullet points eg.
* check out [Hybridize tutorial] ....
* Also check out [custom layers] ...

File: python/guide/getting-started/to-mxnet/pytorch.md [hunk 3]
```diff
@@ -1,15 +1,8 @@
 # MXNet for PyTorch users in 10 minutes
 
-[PyTorch](https://pytorch.org/) has quickly established itself as one of the
-most popular deep learning framework due to its easy-to-understand API and its
-completely imperative approach. But you might not be aware that MXNet includes
-the Gluon API which gives you the simplicity and flexibility of PyTorch, whilst
-allowing you to hybridize your network to leverage performance optimizations of
-the symbolic graph.
+[PyTorch](https://pytorch.org/) is a popular deep learning framework due to its easy-to-understand API and its completely imperative approach. Apache MXNet includes Gluon API which gives you the simplicity and flexibility of PyTorch and also allows you to hybridize your network to leverage performance optimizations of the symbolic graph.
```
Comment (sad-) [inline python/guide/getting-started/to-mxnet/pytorch.md]: suggest rewriting to: "Apache MXNet includes **the** Gluon API .... Pytorch and ~~also~~ allows you to ...."

**Comment by Ishitori** [comment]
Applied all changes that @sad- proposed.

File: python/guide/getting-started/to-mxnet/pytorch.md [hunk 4]
```diff
@@ -1,15 +1,8 @@
 # MXNet for PyTorch users in 10 minutes
 
-[PyTorch](https://pytorch.org/) has quickly established itself as one of the
-most popular deep learning framework due to its easy-to-understand API and its
-completely imperative approach. But you might not be aware that MXNet includes
-the Gluon API which gives you the simplicity and flexibility of PyTorch, whilst
-allowing you to hybridize your network to leverage performance optimizations of
-the symbolic graph.
+[PyTorch](https://pytorch.org/) is a popular deep learning framework due to its easy-to-understand API and its completely imperative approach. Apache MXNet includes the Gluon API which gives you the simplicity and flexibility of PyTorch and allows you to hybridize your network to leverage performance optimizations of the symbolic graph.
```
Comment (thomelane) [inline python/guide/getting-started/to-mxnet/pytorch.md]: Would reference the performance gains you'd expect from using MXNet, and give statistics from NVidia's performance benchmarks. Quite like starting the next sentence with "But" which was removed.

File: python/guide/getting-started/to-mxnet/pytorch.md [hunk 5]
```diff
@@ -54,19 +50,17 @@ y = x + 1
...
+Both frameworks support multiple functions to create and manipulate tensors / NDArrays. You can find more of them in the documentation.
 
 ## Model training
 
-Let's look at a slightly more complicated example below. Here we create a
-Multilayer Perceptron (MLP) to train a model on the MINST data set. We divide
-the experiment into 4 sections.
+After covering the basics of data creation and manipulation, let's dive deep and compare how model training is done in both frameworks. In order to do so, we are going to solve image classification task on MNIST data set using Multilayer Perceptron (MLP) in both frameworks. We divide the task in 4 steps.
 
 ### 1 --- Read data
```
Comment (thomelane) [inline python/guide/getting-started/to-mxnet/pytorch.md]: Would avoid this style of section title. We don't use it anywhere else. Stick with "### 1. Read Data". Same applied for other section headings.

File: python/guide/getting-started/to-mxnet/pytorch.md [hunk 6]
```diff
@@ -205,27 +191,28 @@ for epoch in range(5):
...
 
-* In this sample, MXNet is twice as fast as PyTorch. Though you need to be
-cautious with such toy comparisons.
+* In this sample, Apache MXNet is twice as fast as PyTorch. Though you need to be cautious with such toy comparisons.
 
 ## Conclusion
 
-MXNet and PyTorch share a lot of similarities. It makes PyTorch users much easy
-to use MXNet.
+As we saw above, Apache MXNet Gluon API and PyTorch are similar in use.
```
Comment (thomelane) [inline python/guide/getting-started/to-mxnet/pytorch.md]: Can you summarize the main differences?

File: python/guide/getting-started/to-mxnet/pytorch.md [hunk 7]
```diff
@@ -205,27 +191,28 @@ for epoch in range(5):
...
+
+## Recommended Next Steps
+
+While Apache MXNet Gluon API is very similar to PyTorch, there are some extra functionality using which can make your code even faster.
```
Comment (thomelane) [inline python/guide/getting-started/to-mxnet/pytorch.md]: using which can -> that can

File: python/guide/getting-started/to-mxnet/pytorch.md [hunk 8]
```diff
@@ -1,15 +1,8 @@
 # MXNet for PyTorch users in 10 minutes
```
Comment (thomelane) [inline python/guide/getting-started/to-mxnet/pytorch.md]: PyTorch vs MXNet

**Comment by Ishitori** [comment]
Fixed all described issues.
